<?xml version="1.0"?>
<launch>
  <node name="joy" pkg="joy" type="joy_node" output="screen" />
  <node name="rover_teleop" pkg="casy_rover" type="rover_teleop" output="screen" >
    <param name="x_axis" value="5"/>
    <param name="y_axis" value="4"/>
    <param name="z_axis" value="2"/>
    <param name="yaw_axis" value="1"/>
    <remap from="apm/send_rc" to="send_rc" />
  </node>
</launch>
